.github/workflows/update-otel-cron.yml

```
@@ -0,0 +1,32 @@
+name: 'Update contrib to the latest core source'
+on:
+  schedule:
+    - cron: "27 21 * * *" # Run at an arbitrary time on weekdays.
+
+jobs:
+  update-otel:
+    runs-on: ubuntu-24.04
+    if: ${{ github.repository_owner == 'open-telemetry' }}
```

you don't need `${{}}` in `if:`s
```suggestion
    if: github.repository_owner == 'open-telemetry'
```